index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <div id="logo">
+        <img src="images/bbbafc0f4a1b47ad902423510cde7f1b.png" alt="" />
+      </div>
+      <ul>
```

Comment:
Why don't you use CSS classes rather than targeting the element with an _id_. This is for best practises as _ids_ are targeted using Javascript.